packages/gatsby-link/README.md

```diff
@@ -38,3 +39,17 @@ render () {
   </div>
 }
 ```
+
+## Forcing internal redirect
```

Comment:
The normal term for this is "Programmatic Navigation" — please change this header to that

e.g. https://stackoverflow.com/questions/31079081/programmatically-navigate-using-react-router